Scripts/Items/Books/SpecialScrollBooks/BaseSpecialScrollBook.cs

```diff
@@ -1,182 +1,214 @@
+        {
+            base.GetContextMenuEntries(from, list);
+
+            SetSecureLevelEntry.AddTo(from, this, list);
+        }
+
+        public override bool OnDragDrop(Mobile m, Item dropped)
+        {
+            if (m.InRange(GetWorldLocation(), 2))
+            {
+                BaseHouse house = BaseHouse.FindHouseAt(this);
+
+                if (dropped.GetType() != ScrollType)
+                {
+                    m.SendLocalizedMessage(BadDropMessage);
+                }
+                else if (house == null || !IsLockedDown)
+                {
+                    m.SendLocalizedMessage(1151765); // You must lock this book down in a house to add scrolls to it.
+                }
+                else if (!house.HasSecureAccess(m, Level))
+                {
+                    m.SendLocalizedMessage(1155693); // This item is impermissible and can not be added to the book.
+                }
+                else if (Items.Count < Capacity) // TODO: Message for overfilled?
+                {
+                    DropItem(dropped);
+
+                    m.SendLocalizedMessage(DropMessage);
+
+                    dropped.Movable = false;
+
+                    m.CloseGump(typeof(SpecialScrollBookGump));
+
+                    return true;
+                }
+            }
+
+            return false;
+        }
+
+        public virtual void Construct(Mobile m, SkillName sk, double value)
+        {
+            var scroll = Items.OfType<SpecialScroll>().FirstOrDefault(s => s.Skill == sk && s.Value == value);
+
+            if (scroll != null)
+            {
+                if (m.Backpack == null || !m.Backpack.TryDropItem(m, scroll, false))
+                {
+                    m.SendLocalizedMessage(502868); // Your backpack is too full.
+                }
+                else
+                {
+                    BaseHouse house = BaseHouse.FindHouseAt(this);
+
+                    if (house != null && house.LockDowns.ContainsKey(scroll))
+                    {
+                        house.LockDowns.Remove(scroll);
+                    }
+
+                    if (!scroll.Movable)
+                    {
+                        scroll.Movable = true;
+                    }
+
+                    if (scroll.IsLockedDown)
+                    {
+                        scroll.IsLockedDown = false;
+                    }
+
+                    m.SendLocalizedMessage(RemoveMessage);
+                }
+            }
+        }
+
+        public BaseSpecialScrollBook(Serial serial)
+            : base(serial)
+        {
+        }
+
+        public override void Serialize(GenericWriter writer)
+        {
+            base.Serialize(writer);
+
+            writer.Write(2); // version
+
+            writer.Write((int)Level);
+            writer.Write(_Capacity);
+        }
+
+        public override void Deserialize(GenericReader reader)

```

Comment:
https://github.com/ServUO/ServUO/pull/3759/files#diff-075c2fb9a54d1f04acb1b5d48220a638

Something here is preventing shards with existing powerscroll books, or probably any special scroll book, for loading up without deleting them. Otherwise everything else looks great!